web/src/styles/globals.css

```diff
@@ -0,0 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+  font-family: "Chalkboard SE";
+  src: url("/fonts/ChalkboardSE-Regular.woff2") format("woff2"),
+    url("/fonts/ChalkboardSE-Regular.woff") format("woff");
+  font-weight: normal;
+  font-style: normal;
+  font-display: fallback;
+}
+
+@layer base {
+  body {
+    @apply bg-white text-black dark:bg-black dark:text-white;
+  }
+
+  button:disabled {
+    @apply brightness-50 cursor-not-allowed;
+  }
+
+  h1 {
+    @apply text-[2rem];
```

Comment:
Same with all of these below, let's try not to use `@apply` at all, we'll just inline these

Reply:
I'll remove all of the `@apply` directives